### DISCUSSION TITLE
Welcome to Stellarium Discussions!

### DISCUSSION THREAD

**Discussion by alex-w** [Stellarium/stellarium#1398] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by gzotti** [comment]
Just had my talk at IAUS367, now appeared at https://www.youtube.com/channel/UCrDZ4rKiYSzGc7ZfN9_OcCA/videos
- Many nice reactions. 
- Suggestion to implement: Galactic center marker.

**Reply by alex-w** [reply]
OK, l’ll add galactic center marker.

**Reply by gzotti** [reply]
:-)

**Reply by alex-w** [reply]
I’ve added galactic center and galactic anticenter points

**Comment by ghost** [comment]
Mine is a very simple question I almost have to be ashamed of. So far I have used Stellarium very seldomly. But now that I have a telescope it is planned to be increased.

When opening Stellarium today and looking for an update I got to [https://github.com/Stellarium/stellarium/releases/v0.20.3](url)
Then I clicked on a small input field:
   downloads @ V0.20.3.  [black]  273 k  [green]
Then nothing visible for me happened: in the Firefox download directory no new file

1. What was my mistake, where to find the announced tar.gz file?
2. Would it be possible to change the page a little bit so that newbies like me would have easier access to updates?

A. Mehl

PS Many thanks to all the people making possible this wonderful program.

**Reply by gzotti** [reply]
Just scroll down the page to find all downloads.

**Reply by ghost** [reply]
Thank you for your reply. BUT the small field I mentioned in my first post is clickable. So I deem it obvious to click there. Why not improve this page by stating e.g. 'The updates can be found at the buttom of the page'?  In addition a very short -  three to five lines - instruction on how to install this would be welcome. Not everybody installs programs from the terminal; I do it normally using rpm's and will have to look it up how to install.

Thanks again for your work for the community.

A. Mehl

**Reply by silas1037** [reply]
@arnozorro install guide is in PDF guide.

**Comment by silas1037** [comment]
Excellent. When will 0.20.4 publish?

**Reply by gzotti** [reply]
soon.

**Comment by axd1967** [comment]
@gzotti is there a heads-up procedure to warn the translators of the upcoming release? Translators need some time to check what needs to be translated, what has been added, etc.

**Reply by gzotti** [reply]
@alex-w ? Translators should be aware of the 4-releases-per-year cycle. There is no more direct call I am aware of. They should work in between to avoid high workloads in the last days.

**Reply by alex-w** [reply]
Sorry, we did not send a special notifications for translators in last few years (after switch to new release cycle)

**Reply by axd1967** [reply]
Until now I was not watching the Stellarium Transifex project, that's maybe why I was wondering how I would be informed of changes. I assume I will get some sort of notification when new source strings are available for translation.

Maybe some form of message should be sent via Transifex to notify translators of an upcoming release, and some time foreseen for them to complete the translations? Until now this part of the process flow is not clear to me - I have the impression that my translations will always end up in "a next release" rather than in the upcoming release.

_edit added: https://docs.transifex.com/localization-tips-workflows/content-update-translation_

**Reply by axd1967** [reply]
I just see this:

> Collaborators who didn't enable the Watch feature will still be notified when a new resource is created in Transifex. 

I cannot recall having had such a notification (eg for the Calendar plugin).

https://docs.transifex.com/tracking/notifications#watch-notifications

**Reply by axd1967** [reply]
This might be handy for notifying the translators:?
https://docs.transifex.com/teams/announcements-and-discussions

**Reply by alex-w** [reply]
> Until now I was not watching the Stellarium Transifex project, that's maybe why I was wondering how I would be informed of changes. I assume I will get some sort of notification when new source strings are available for translation.

OK, you disable notifications but you want to get these notifications. I think this problem has an obvious solution.

> Maybe some form of message should be sent via Transifex to notify translators of an upcoming release, and some time foreseen for them to complete the translations? 

Maybe. But Stellarium has a "fixed" dates of an upcoming releases - do you still need a separate notifications?

> Until now this part of the process flow is not clear to me - I have the impression that my translations will always end up in "a next release" rather than in the upcoming release.

We are apply all exist translated data on date on release. If you will translate some data after release, then it will be up in "a next release" of course.

**Reply by alex-w** [reply]
> I just see this:
> 
> > Collaborators who didn't enable the Watch feature will still be notified when a new resource is created in Transifex.
> 
> I cannot recall having had such a notification (eg for the Calendar plugin).
> 
> https://docs.transifex.com/tracking/notifications#watch-notifications

Adding a new resource is not the same as updating a source file in an existing resource.

**Reply by axd1967** [reply]
> Maybe. But Stellarium has a "fixed" dates of an upcoming releases - do you still need a separate notifications?

Ok, found back https://github.com/Stellarium/stellarium/wiki/Release-Schedule.

The issue is this: 

>  usually around the solstices and equinoxes

Is this a matter of hours? days? weeks? 

the release flow needs some time devoted to translations, it is not clear to me if this has been taken into account; one cannot expect translators to have everything translated at the last minute.

Allow me to suggest to make this into hard dates ("In the week containing solstice/equinox") , so there is no ambiguity / uncertainty. Translators would receive e.g. 1 week extra before a release. This makes it clear for everyone, and is a nice deadline.

**Reply by gzotti** [reply]
Given that Alex is building basically weekly betas with updates strings, regard translations as "rolling release" matter. Missing translations don't stop a release, and are a nicety at will of the translators. Sure, the calendar plugin was a recent, very recent addition, the alternative would have been delaying it to 0.21.0. Just look into Transifex once a month, and your language will be mostly complete. You cannot impose hard deadlines on a project that basically exploits developers' vacations.

**Reply by alex-w** [reply]
> The issue is this:
> 
> > usually around the solstices and equinoxes
> 
> Is this a matter of hours? days? weeks?

We publish releases in the solstices and equinoxes, but release may be delayed on few days due fixing critical issues or missing critical data. You may see it when you see the list of the dates of releases in last few years. This is obvious.

> the release flow needs some time devoted to translations, it is not clear to me if this has been taken into account; one cannot expect translators to have everything translated at the last minute.

We expect that translators will do their work to complete up to the solstices and equinoxes to allow us to merge translations. Yes, the upcoming release got many lines for translation in "last minute", but we delayed release also.
 
> Allow me to suggest to make this into hard dates ("In the week containing solstice/equinox") , so there is no ambiguity / uncertainty. Translators would receive e.g. 1 week extra before a release. This makes it clear for everyone, and is a nice deadline.

Sorry, but if day of solstice/equinox is not a "hard date" for you, then I cannot help you.

**Reply by alex-w** [reply]
I'm forget to say that we have a planned milestones for releases at minimum on future half year: https://github.com/Stellarium/stellarium/milestones

**Comment by paolo1953** [comment]
Grazie
Devo capire come si fa a rispondere.....  al benvenuto....

**Comment by paolo1953** [comment]
Buongiorno, credo di aver chiesto aiuto anche tempo fa per questo argomento...., ma Stellarium mi piace cosi tanto che ritento...
Riprovo a parlare di Stellarium che non funziona con Eagle4 di primaluce lab
ho un paio di montature Celestron Avx  e una CGX...., con collegamento diretto al pc nessun problema stellarium funziona benissimo collego la montatura e si vede subito la posizione del telescopio basta premere i tasti di puntamento e il telescopio si muove....... Ho acquistato dalla ditta Primaluce lab  i mini pc Eagle 4 e Eagle 4S per la gestiune dei dati e dell'alimentazione   conon la promessa che avrebbe funzionato anche Stellarium..... ma non è stato cosi...,      riprovato più volte ma non ha mai funzionato......., Non riesco a capire cosa faccio di sbagliato o cosa devo settare per farlo funzionare........., chissà se qualcuno ha avuto il mio stesso problema...?
Grazie
saluti
                                            Paolo

**Comment by sjhanc** [comment]
I just started using the program, I love it. I have a question concerning the mouse. On my PC, when I am running the program, the mouse cursor is invisible. This makes navigating very difficult. How do I make the cursor visible?  When I drag the mouse to the lower left corner, the tool bar appears and when the invisible cursor is in the area of a control, its icon is highlighted, but still no mouse cursor. Keyboard commands work as they should. The cursor works normally on all other programs.

**Reply by sjhanc** [reply]
I figured it out, take it out of full screen.

**Reply by gzotti** [reply]
Please see the FAQ or "common problems" in the wiki about "Mouse Cursor Invisible".

**Comment by jbsoundideas** [comment]
I am helping out to establish a new children's museum in Kona Hawaii.

I want to use a touchscreen to enable youth ages 67-10 to explore Hawaiian
night skies with Stellarium.

Is there a way to set it up with only selected control icons on the
touchscreen?

Since the kids will only be there in the daytime is there a way to
fix/limit the times for evening only but at this date and location?

Is there a way to have a scroll bar at the bottom of the screen to change
times or dates?

**Reply by gzotti** [reply]
Not as built-in features. However, there is the RemoteControl plugin. See the User Guide, section 13.5. You could develop your own restricted user interface. Make a copy of index.html and remove controls you don't need. The advantage is: it runs in a web browser on a second control screen. If you need to still exploit touch interaction with the main screen, switch off the main menu. (Ctrl-T)

**Comment by jbsoundideas** [comment]
Wow. Great info.

Any thought on setting the times to evening only as default?


-- 

Jeff Bowen F-IPS, F-GLPA

73-1234 Hamo Street
Kailua-Kona, HI 96740
317 840 4802

  "Slow down! This isn't the mainland." (Big Island license plate😉.

**Reply by gzotti** [reply]
You can configure a time the program will show at launch. But for a program that should run the whole day, freezing the time sounds weird to me, as progress of time is an inherently important part of any "sky simulation".  In your own interface, you could make some sort of "tonight" button which calls a little 1-line script, or you could even configure a cron job that calls Stellarium via HTTP API and triggers that script to reset the date/time every ten minutes or so. However, this may turn out more annoying than useful.

**Comment by jbsoundideas** [comment]
I agree but there will not be any kids there after 5 so I want them to 
learn what to look for at night.

Similar to visiting a real planetarium in daytime.

-- 

Jeff Bowen F-IPS, F-GLPA

73-1234 Hamo Street
Kailua-Kona, HI 96740
317 840 4802

  "Slow down! This isn't the mainland." (Big Island license plate😉.

**Reply by gzotti** [reply]
If you really only want to show the sky when the sun is e.g. 12° below the horizon (end of Nautical twilight), configure this altitude in the regular GUI (Figure 4.10, Solar altitude for Twilight Finder"), jump to "Today's evening twilight". See User Guide, section 4.8.1. Then stop time progress and hide user interface, and don't show time controls in the web GUI that you will develop.

**Reply by gzotti** [reply]
You can also call this functionality in your startup script.

**Comment by jbsoundideas** [comment]
Hmmmmm...

Jeff Bowen
Fellow-IPS, Fellow-GLPA
73-1234 Hamo Street
Kailua-Kona, HI 96740
317 840 4802

**Comment by BestCoder1877** [comment]
UI could be very clunky and could be improved otherwise is really advanced and better then night sky

**Comment by rowan9999** [comment]
Hello, this is my 1st post. For the solar eclipse on the 12th, can anyone tell me where/how i find  out the angle of elevation above the horizon, and the compass bearing of the sun at eclipse time, from different locations? Thank you very much indeed, especially if the answer is really really clear and easy for me to understand, as i'm getting very very tired in last few days of hectic prep for a cycle touring trip to Spain! Thank you, Rowan

**Reply by gzotti** [reply]
Stellarium First Contact: Set date, time, click on the sun and read altitude (degrees above mathematical horizon) and azimuth (compass bearing). Use the Location panel to set your location. The rest is described in the User Guide.

**Comment by rowan9999** [comment]
Thank you! I've tried to find  Stellarium First Contact, but can't- where
is it?

**Reply by gzotti** [reply]
I just provided this ad-hoc session. The User Guide has an introductory chapter you may want to read.

**Comment by rowan9999** [comment]
Hi Georg, i've tried to find the user guide but can't. I really don't
understand how the Stellarium website works. I find it very odd. Thank you!

**Reply by gzotti** [reply]
Seriously? You managed to create an account here, so you presumably can operate a PC. Open your web browser and type "Stellarium User Guide". Press Enter. Follow any of the links displayed that seem meaningful.

**Comment by rowan9999** [comment]
thanks, got that now.
I was just getting lots of odd stuff to do with computer programming before
